### ISSUE TITLE
Some questions about benchmark in paper

### ISSUE THREAD

**Issue by zyfone** [AmingWu/Single-DGOD#3]
First of all, thank you very much for setting set by the author
I have completely reproduced many baseline by using the author's data set
The results are almost consistent with the author's description, which makes me feel happy and admires the author's scientific research level
The author's many contributions will be beneficial to the development of the community

But  there are still some problems in the paper, which I don't quite understand、
As a freshman, the problem may be a little low-level. I hope the author can be a little tolerant
Q:
1.feature normalization methods are a litte less than baseline（faster-rcnn）  Is there no need for comparison？
2. In Ablation Analysis，Why do test on the training set？（Daytime-Sunny）
3. Decoupling in previous articles of the author usually takes the way of updating parameters step by step，however，In this article, an updated way is adopted  as same as DDF(Decompose to Adapt: Cross-domain Object Detection via Feature Disentanglement).
In my view,according to  experiment which only one data set is trained, and then the expectation and variance on convolution between different data sets are observed。I found that on two data sets with domain shift, the trained backbone was not consistent with the data distribution of the following layers。In the nutshell, **shallow decoupling information will still introduce domain shift after some  convolution？**
Is the next development of decoupling method to solve the problem of preventing shallow information from introducing domain shift？

Last but no least，thanks to the author for his contribution to the domain adaptive community and his pioneering approach to feature decoupling

**Comment by AmingWu** [comment]
Thank you. 
(1) Since feature normalization is a commonly used method to improve generalization, it is better to compare with it.
(2) I consider that a generalized model should own superior performance on new domain while maintaining the performance on the training domain. Thus, we also evaluate the performance on the training domain.
(3) The shallow information is also important for boosting generalization. Thanks. I will try to use the shallow information.

**Comment by asuweb0** [comment]
still waiting for the code

**Comment by AmingWu** [comment]
Thank you. I will release the code next week.

**Comment by SnowdenLee** [comment]
Hi @AmingWu, 
is there any updates on the code release? Thanks a lot!
